### DISCUSSION TITLE
输入nb run出现UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc5 in position 80: invalid continuation byte

### DISCUSSION THREAD

**Discussion by bentwotwo** [nonebot/discussions#167] [问答]
![image](https://user-images.githubusercontent.com/107903182/174710375-4b17d194-5874-4a53-8851-35b148b85130.png)

![image](https://user-images.githubusercontent.com/107903182/174710259-e3d701a7-2f83-465a-800c-81c46a52d62d.png)

**Comment by yanyongyu** [comment]
请查看一下你的`.env`以及`.env.*`文件编码是否为utf-8

**Reply by bentwotwo** [reply]
弄好了，感谢
